2020/10/2020-10-23-RIAA.md

```diff
@@ -1,118 +1,119 @@
-
-We also note that the source
-code prominently includes as sample uses of the source code the downloading of copies of our
-members’ copyrighted sound recordings and music videos, as noted in Exhibit A hereto. For
-example, as shown on Exhibit A, the source code expressly suggests its use to copy and/or
-distribute the following copyrighted works owned by our member companies:
-
-• Icona Pop – I Love It (feat. Charli XCX) [Official Video], owned by Warner Music Group  
-• Justin Timberlake – Tunnel Vision (Explicit), owned by Sony Music Group  
-• Taylor Swift – Shake it Off, owned/exclusively licensed by Universal Music Group
-
-The source code notes that the Icona Pop work identified above is under the YouTube Standard
-license, which expressly restricts access to copyrighted works only for streaming on YouTube
-and prohibits their further reproduction or distribution without consent of the copyright owner;
-that the Justin Timberlake work identified above is under an additional age protection
-identifier; and that the request for the Taylor Swift work identified above is to obtain, without
-authorization of the copyright owner or YouTube, an M4A audio file from the audiovisual work
+Deaw Siw ow Madam (* ^ ω ^):
+
+I am contacting you on behawf of the Wecowding Industwy Association of Amewica, Inc. (WIAA) and
+its membew wecowd companyies (\*/ω＼). The WIAA is a twade association whose membew companyies
+cweate, manyufactuwe ow distwibute sound wecowdings wepwesenting appwoximatewy eighty-five (85)
+pewcent of aww wegitimate wecowded music consumption in the Unyited States (；￣Д￣). Undew penyawty of
+pewjuwy, we submit that the WIAA is authowized to act on behawf of its membew companyies on
+mattews invowving the infwingement of theiw sound wecowdings, audiovisuaw wowks and images,
+incwuding enfowcing theiw copywights and common waw wights on the Intewnyet (＃\`Д´).
+
+Copywight Viowations. We have weawnyed that youw sewvice is hosting the youtube-dw souwce code
+on its nyetwowk at the fowwowing wocations, among othews ヽ( \`д´*)ノ:
+
+https://github.com/ytdw-owg/youtube-dw  
+https://github.com/ytdw-owg/youtube-dw/twee/gh-pages  
+http://ytdw-owg.github.io/youtube-dw/  
+https://github.com/benkeung/youtube-dw  
+https://github.com/cybewjacob/youtube-dw  
+https://github.com/ewaopinska/youtube-dw  
+[pwivate]    
+https://github.com/huangciyin/youtube-dw  
+https://github.com/jckewwey/youtube-dw  
+https://github.com/WouisPwisso/youtube-dw  
+https://github.com/ojauch/youtube-dw  
+https://github.com/wbwito/youtube-dw  
+https://github.com/successWee/youtube-dw  
+https://github.com/twammew/youtube-dw  
+https://github.com/vs9390/youtube-dw  
+https://github.com/zackfewn/youtube-dw  
+https://github.com/tosuch/youtube-dw  
+https://github.com/pownyophage/youtube-dw  
+https://github.com/tejaskhot/youtube-dw  
+https://github.com/VideoUtiws/youtube-dw  
+https://github.com/798221028/youtube-dw  
```

Comment:
URLs should obviously not be changed tho

Reply:
No they should

Reply:
considering a URL is a location pointer they should point to the correct locations. whether or not the names of the original locations should be changed is its whole own thing.

Reply:
Then we shouwd change them to some kawaii anyime music (´ ε ` )♡